### DISCUSSION TITLE
Holding 'Random' button in Song Select replays the randomization sound which can be pretty loud and obnoxious

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#16803] [Q&A]

**Comment by bdach** [comment]
was a deliberate choice at time of implementation as far as i can tell (see https://github.com/ppy/osu/pull/16675#discussion_r797456145)

**Reply by peppy** [reply]
The eventual plan would be to have this behave similar to stable, where holding enters an ongoing animation until the key is released.
